### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by Scotchester**
- Update with cleared content
- Add hero illustration
- Reformat pre-footer stuff into sidebar

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -15,100 +15,196 @@
...
-{% block content_main_modifiers -%}
-    content__flush-bottom
-{%- endblock %}
-
 {% block content_main %}
 
-    <h2>Financial savings tools</h2>
+    <div class="m-image-text-25-75">
+        <div class="m-image-text-25-75_image-container">
+            <img class="m-image-text-25-75_image" src="http://placehold.it/300">
```
Comment (jimmynotjim) [inline src/index.html:33]: This placeholder image should be updated.

File: src/index.html [hunk 2]
```diff
@@ -15,100 +15,196 @@
...
+        <div class="m-image-text-25-75_content">
+            <h2>Financial savings tools</h2>
+            <p>
+                You’re not alone when saving for your future.
+                Check out our tools and resources that can help you make the most
+                of your savings and achieve your financial goals:
+            </p>
+            <ul>
+                <li>
+                    Planning for retirement?
```
Comment (jimmynotjim) [inline src/index.html]: Should these headings be wrapped in a `h#` elem?

File: src/index.html [hunk 3]
```diff
@@ -15,100 +15,196 @@
...
+            <p>
+                Thinking about starting a retirement account?
+                The Department of the Treasury has a new
+                retirement savings account option called myRA.
+                Here’s what makes myRA a good savings option:
+            </p>
+
+            <ul>
+                <li>
+                    <b>It’s affordable.</b>
```
Comment (jimmynotjim) [inline src/index.html:80]: These should prob be converted to `h#` rather than `b` elem

File: src/index.html [hunk 4]
```diff
@@ -15,100 +15,196 @@
...
-                           href="https://www.linkedin.com/company/consumer-financial-protection-bureau">
-                            <span class="cf-icon cf-icon-linkedin-square"></span>
-                            <span class="u-visually-hidden">LinkedIn</span>
-                        </a>
-                    </li>
-                </ul>
-            </aside>
+    <div class="m-image-text-25-75">
+        <div class="m-image-text-25-75_image-container">
+            <img class="m-image-text-25-75_image" src="http://placehold.it/300">
```
Comment (jimmynotjim) [inline src/index.html:124]: Update this placeholder as well.

**Comment by Scotchester** [comment]
Thanks for the review, @jimmynotjim! I probably should have given a little more context. The illustrations that have placeholders are still being completed, so the placeholders are fine for now.

I'll give the ideas for additional heading usage some consideration. On the ones that are currently `b` elements, those are inline with the following sentence. We don't have any utility classes or mixins to handle combining two block elements into an inline display, do we?

**Comment by jimmynotjim** [comment]
I don't believe we do. Should they be `strong` instead of `b` or is it really just a visual change?

**Comment by Scotchester** [comment]
I think the `b`s should remain `b`s and the other ones above it that you called out should be converted to headings.

**Pull request merged**